src/SDKs/Graph.RBAC/Graph.RBAC/Microsoft.Azure.Graph.RBAC.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Graph.RBAC</PackageId>
-    <VersionPrefix>3.4.0-preview</VersionPrefix>
+    <VersionPrefix>3.5.0-preview</VersionPrefix>
     <Description>Microsoft.Azure.Graph.RBAC</Description>
     <AssemblyName>Microsoft.Azure.Graph.RBAC</AssemblyName>
```

Comment:
Please add `PackageReleaseNotes` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/600ad721aad9e1a8831d293619ce1be09aef6d7d/src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj#L11)